en/docs/security-announcements/cve-justifications/2025/CVE-2021-42646.md

```diff
@@ -0,0 +1,53 @@
+---
+title: CVE-2021-42646
+category: security-announcements
+date: June 02, 2025
+---
+
+# CVE-2021-42646
+
+<p class="doc-info">WSO2 Products Impacted: No</p>
+<p class="doc-info">Customers Actions Required: No</p>
+---
+
+### REPORTED VULNERABILITY
+An XML External Entity (XXE) vulnerability in the file-based service provider creation feature of the WSO2 Management Console allows attackers to send crafted GET requests to access sensitive files or cause denial of service by exploiting insecure XML parsing [^1].
+
+### REPORTED PRODUCTS
+- WSO2 API Manager: 2.6.0, 3.0.0, 3.1.0, 3.2.0, 4.0.0  
+- WSO2 Identity Server: 5.7.0, 5.8.0, 5.9.0, 5.10.0, 5.11.0  
+- WSO2 Identity Server as Key Manager: 5.7.0, 5.9.0, 5.10.0
+- WSO2 Open Banking KM: 1.3.0, 1.4.0, 1.5.0
```

Comment:
@Tharanidk let's remove the OB versions since those are not in the advisory

Reply:
sure

Reply:
sure